nautobot/extras/tables.py

```diff
@@ -761,11 +761,12 @@ class TagTable(BaseTable):
     pk = ToggleColumn()
     name = tables.LinkColumn(viewname="extras:tag", args=[Accessor("slug")])
     color = ColorColumn()
+    content_types = ContentTypesColumn(truncate_words=15)
```

Comment:
15 seems like a lot to me - does this render reasonably in the case where a tag has many/all content-types associated?

Reply:
![Screenshot 2022-04-08 at 6 48 25 PM](https://user-images.githubusercontent.com/94907097/162494964-f2c5e223-9c05-4c90-8665-24360ee6eea5.png)